src/aria/widgets/form/list/ListController.js

```diff
@@ -499,8 +499,10 @@ module.exports = Aria.classDefinition({
         calcMoveFocus : function (flowOrientation, focusIndex, numberOfRows, numberOfColumns, keyCode, numberItems) {
 
             var moveFocus = focusIndex;
+            var cfg = this._data.cfg;
+            var waiAria = cfg && cfg.waiAria;
 
-            if (numberOfColumns == 1) {
+            if (numberOfColumns == 1 || waiAria) {
```

Comment:
In fact, I have just tried with JAWS, and it seems to work fine even with multiple columns... so I think it is possible to even remove the line is the `$constructor` of the `List` class.